lib/client/src/client.rs

```diff
@@ -515,15 +481,21 @@ impl MangoClient {
         account: &MangoAccountValue,
         market_index: Serum3MarketIndex,
         side: Serum3Side,
-        price: f64,
-        size: f64,
+        limit_price: u64,
+        max_base_qty: u64,
+        max_native_quote_qty_including_fees: u64,
```

Comment:
There definitely should be a simple ix constructor like this one; but wondering if we should keep mc's idea of having a simplified one with ui numbers. 

I guess this client is for advanced users who're ok figuring out "quote_qty_incl_fees" themselves. 

I'd like if it was `limit_price_lots` and `max_base_qty_lots` to be clearer about the units of these two.

Reply:
ui numbers should be another layer on top, ideally a rest api